tests/cpp/dlr_pipeline_skl_xgb_test.cc

```diff
@@ -156,5 +156,7 @@ int main(int argc, char** argv) {
 #ifndef _WIN32
   testing::FLAGS_gtest_death_test_style = "threadsafe";
 #endif  // _WIN32
+  // temporarily disable tests (TO DO: generate new artifacts)
```

Comment:
Lets use `TODO` (without space in the middle) - so we can easily find all TODOs if we search for TODO

Reply:
Done